evasquez26/angular4-startup
# S6-ONE-261 Fixed when trying to retrieve intructors ids from course when they are empty

## Story / Task

[ONE-261](https://bixlabs.atlassian.net/browse/ONE-261)

## What this PR does?

Fixes an error when trying to view a course details and the instructors ids are empty.


### Before 

- We could not see course details when the course did not have instructors, somehow at the beginning we created courses without instructor ids

### After

- We can see course details even if they do not have instructors (which they should not, at some point we started passing as default the id of the user who creates the course)
- We pass user id who creates the course as default when updating and the course has no instructors (also notice the API has a problem when updating the course and we try to add new instructors, is currently being reviewed by Lev)

### How to test / reproduce

- Navigate to explore
- Click math category
- Select any course of the bottom list, because they were the first created without validations
- Observe that the course details opens without problems
- Observe that the teachers field is empty (for now)

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

 Thanks!
